data-raw/sao-paulo-activity-data.R

```diff
@@ -9,7 +9,7 @@ unlink("OD-2017", recursive = T, force = T)
 
 # just São Paulo municipality with short trips
 OD_SP_2017 = OD_SP_2017 %>%
-  filter(MUNI_DOM == 36 & DISTANCIA <= 5000)
+  filter(ZONA_O %in% c(1:9, 18:19, 24:26) & ZONA_D %in% c(1:9, 18:19, 24:26))
```

Comment:
I think we should include a version with fewer columns. I can add that post merge.